test/test_projects/c.py

```diff
@@ -10,15 +10,16 @@
 {{ spam_c_top_level_add }}
 
 static PyObject *
-spam_system(PyObject *self, PyObject *args)
+spam_filter(PyObject *self, PyObject *args)
 {
-    const char *command;
+    const char *content;
     int sts;
 
-    if (!PyArg_ParseTuple(args, "s", &command))
+    if (!PyArg_ParseTuple(args, "s", &content))
         return NULL;
 
-    sts = system(command);
+    // Spam should not be allowed through the filter.
+    sts = strcmp(content, "spam");
```

Comment:
`system()` is a prohibited call on iOS (there's essentially no access to spawn subprocesses). However, this didn't seem to be what was actually being tested, so I've converted it into a simpler "string check" with the name net result (a 0/1 integer result that is controlled by input)

Reply:
Yeah, that's fine. I think really all we care about is that it can call a C library function here.